approval/service.go

```diff
@@ -86,6 +86,9 @@ func (s *service) AdvanceApproval(appeal *domain.Appeal) error {
 
 				if passed {
 					approval.Status = domain.ApprovalStatusApproved
+					if i+1 <= len(appeal.Approvals)-1 {
+						appeal.Approvals[i+1].Status = domain.ApprovalStatusPending
+					}
 				} else {
 					if stepConfig.AllowFailed {
 						approval.Status = domain.ApprovalStatusSkipped
```

Comment:
```suggestion
						approval.Status = domain.ApprovalStatusSkipped
						if i+1 <= len(appeal.Approvals)-1 {
							appeal.Approvals[i+1].Status = domain.ApprovalStatusPending
						}
```
this would be needed in here as well

Reply:
Corrct! This should be here as well! Thanks!